etw/etw.go

```diff
@@ -0,0 +1,7 @@
+package etw

```

Comment:
This really should be in `pkg/etw` to align with golang recommendations

Reply:
Yes I know none of the other packages in this repo adhere to this :)

Reply:
ETW can be the trailblazer :)